changelog/issue-1260.md

```diff
@@ -0,0 +1,33 @@
+level: major
+reference: issue 1260
+---
+
+Google provider in worker-manager now requires you to manually set up
+a service account for your workers to run under. If you are migrating
+from a previously deployed worker-runner, you can just use the account
+we created for you automatically before. Your config will change
+in the following way:
+
+### Old
```

Comment:
I think these are going to look funny in the generated markdown, since each changelog entry is just concatenated to the next.  Maybe `*Old*` and `*New*`?